src/tools/rbd/Utils.h

```diff
@@ -143,7 +143,9 @@ int get_pool_group_names(const boost::program_options::variables_map &vm,
 			 size_t *spec_arg_index,
 			 std::string *pool_name,
 			 std::string *group_name,
-                         std::string *snap_name);
+                         std::string *snap_name,
+                         SnapshotPresence snapshot_presence,
+                         SpecValidation spec_validation);
```

Comment:
@dillaman sorry for offline for weeks. I didn't get your meaning of force proper naming.

Reply:
There is no need to disable validation of the spec / snapshot name -- thus you shouldn't need to pass ```SpecValidation``` to these methods since you should just assume ```SPEC_VALIDATION_FULL```